### ISSUE TITLE
10.4.1: Failed to send action "GetPositionInfo" to [LG] webOS TV

### ISSUE THREAD

**Issue by esabol** [UniversalMediaServer/UniversalMediaServer#2450]
Seeing a lot of this error in the logs:
```
13:36:59 ERROR Failed to send action "GetPositionInfo" to [LG] webOS TV OLED65C9PUA: Error: Current state of service prevents invoking that action. Connection error or no response received.
```
Looking in the full debug logs, I see this:
```
ERROR 2021-03-31 13:36:59.105 [UPNP-[LG] webOS TV OLED65C9PUA] Failed to send action "GetPositionInfo" to [LG] webOS TV OLED65C9PUA: Error: Current state of service prevents invoking that action. Connection error or no response received.
TRACE 2021-03-31 13:36:59.105 [UPNP-[LG] webOS TV OLED65C9PUA] 
org.fourthline.cling.model.action.ActionException: Current state of service prevents invoking that action. Connection error or no response received.
	at org.fourthline.cling.protocol.sync.SendingAction.invokeRemote(SendingAction.java:77)
	at org.fourthline.cling.protocol.sync.SendingAction.executeSync(SendingAction.java:63)
	at org.fourthline.cling.protocol.sync.SendingAction.executeSync(SendingAction.java:51)
	at org.fourthline.cling.protocol.SendingSync.execute(SendingSync.java:54)
	at org.fourthline.cling.protocol.SendingAsync.run(SendingAsync.java:54)
	at org.fourthline.cling.controlpoint.ActionCallback.run(ActionCallback.java:151)
	at net.pms.network.UPNPControl.send(UPNPControl.java:737)
	at net.pms.network.UPNPControl.getPositionInfo(UPNPControl.java:882)
	at net.pms.network.UPNPControl$Renderer.lambda$0(UPNPControl.java:205)
	at java.base/java.lang.Thread.run(Unknown Source)
```
Any ideas?

**Comment by valib** [comment]
@esabol it means that your LG TV doesn't support it. I made the PR #2366 which could fix it.

**Comment by esabol** [comment]
Fwiw, I think it’s likely that my TV was on, but the DLNA app on the TV wasn’t running at the time.

**Comment by valib** [comment]
@esabol can you test it running?

**Comment by esabol** [comment]
I do whenever I watch anything via UMS, obviously.

**Comment by valib** [comment]
Fixed in #2366 and merged to master branch.

**Comment by esabol** [comment]
Wonderful! Thanks, UMS team!
